01_Introduction/Hello/HelloConsole/Program.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Linq;
+
+namespace HelloConsole
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            if (args.Any())
+            {
+                string name = args[0];
+                System.Console.WriteLine($"Hello, {name}!");
```

Comment:
In most cases it doesn't really look nice when you prefix class names with their namespaces, when you can easily avoid it by simply referencing your namespace with `using System;`.

Reply:
Yes, but the project could not build when I used "using System;" and "Console.WriteLine()". I got the error. I chose this way to prevent it.

Reply:
This is very unusual. Did you investigate this? I see that after the updates you are using `using System;` with no issues and it seems to be working now.

Reply:
Yes, this is because I used another PC for the last updates